web/src/components/shared/Trader/Trader.tsx

```diff
@@ -16,36 +20,76 @@ export const Trader: FC = () => {
   const { lang } = useAppContext();
   const { params } = useAdapterContext<UrlParams>();
   const { data = [], error, isValidating } = useUserAds({ publicName: params.name, lang });
+  const [singleMode, setSingleMode] = useState('all');
 
   if (error) {
     return null;
   }
 
+  const onSingleMode = (name: string) => {
+    setSingleMode(name);
+  };
+
   return (
-    <Container maxWidth="fullhd">
-      <Box display="flex" p="8x" height="full">
+    <Container maxWidth="fullhd" className={s.container}>
+      {singleMode === 'all' && (
+        <Box
+          display="flex"
+          p="8x"
+          height="full"
+          flexDirection={{ desktop: 'row', mobile: 'column' }}
+        >
+          <Box
+            display="flex"
+            borderRadius="1.5x"
+            marginRight={{ desktop: '6x' }}
+            px={{ desktop: '6x' }}
+            py={{ desktop: '5x' }}
+            className={s.leftBlock}
+          >
+            <TraderInfo publicName={params.name} onSingleMode={onSingleMode} />
```

Comment:
```suggestion
            <TraderInfo publicName={params.name} onSingleMode={setSingleMode} />
```
Можешь сразу использовать

Reply:
это будет удалено, когда добавлю отдельные роуты